partials/opublikuj-projekt-na-github.md

```diff
@@ -0,0 +1,37 @@
+# Opublikuj projekt na GitHub
+
+## GitHub Desktop
+
+1. Naciśnij `Add a local repository`
+2. Jako `Local Path` wpisz lokalizację folderu z projektem
+3. Naciśnij `create a repository`
+4. Naciśnij `Create Repository`
+5. Naciśnij `Publish repository`
+6. Naciśnij `GitHub.com`
+7. Upewnij się, że `Keep the code private` jest **odznaczone**
+8. Naciśnij `Publish repository`
+9. Twoje repozytorium jest dostępne na `https://github.com/twoja_nazwa_uzytkownika/WARSAWJS_REPO_NAME`
+
+## Wiersz poleceń
+
+1. Przejdź do folderu z twoim projektem 
+    - `cd nazwa_folderu` – komenda do zmiany folderu
+    - `ls` – wypisanie zawartości folderu (macOS/linux)
+    - `dir` – wypisanie zawartości folderu (Windows)
+2. Wykonaj
+
+    ```bash
+    git init
+    git add .
+    git commit -m "Initial commit"
+    ```
+
+3. Stwórz repozytorium na GitHub o nazwie `WARSAWJS_REPO_NAME`
+4. Wykonaj
+
+    ```bash
+    git remote add origin https://github.com/twoja_nazwa_uzytkownika/WARSAWJS_REPO_NAME
```

Comment:
A jeszcze ostatnia rzecz. Jak ugryźć `WARSAWJS_REPO_NAME`? Tj. co pisać zamiast tego, bo ten co był na spodzie teraz już nie będzie działał

Reply:
Dlaczego nie będzie działał?
Ja bym wydzielił ten kawałek kodu do zewnętrznego pliku i injectował na samej górze pliku *.md

Reply:
Bo teraz mamy odniesienia do oddzielnych plików i użytkownik otwiera je z listy w np. `/workshop/21/` i przekieruje go do `/workshop/partials/cra`, a ten js ciągnął nazwę repo z urla.

Reply:
Aaa kumam. W takim razie nic z tym nie zrobimy :/

No i aby było konsekwetnie to bym ten skrypt usunął, bo raz ludzie zobaczą nazwę docelową a raz pattern `WARSAWJS_REPO_NAME`

Reply:
No właśnie usunąłęm już i nie wiem czy zostawić `WARSAWJS_REPO_NAME` czy jakoś to inaczej, ładniej nazwać.